static/app/views/performance/transactionSummary/transactionSpans/suspectSpanCard.tsx

```diff
@@ -211,6 +203,43 @@ function SpanCount(props: HeaderItemProps & {totalCount?: number}) {
   return <HeaderItem label={t('Frequency')} value={value} align="right" />;
 }
 
+function TotalCumulativeDuration(props: HeaderItemProps) {
+  const {sort, suspectSpan, totals} = props;
+
+  let value = (
+    <PerformanceDuration abbreviation milliseconds={suspectSpan.sumExclusiveTime} />
+  );
+
+  if (defined(totals?.sum_transaction_duration)) {
+    value = (
+      <Tooltip
+        title={tct('[percentage] of the total transaction duration of [duration]', {
```

Comment:
Yeah I think using a % against the sum of duration will work for now, we should check how this tooltip looks for http spans on js projects before EA though.

Reply:
Going to leave this like this for now, but yes we'll want to reassess this prior to EA